DESCRIPTION

```diff
@@ -0,0 +1,84 @@
+Package: cfr
+Title: Estimate Disease Severity and Under-Reporting
+Version: 0.1
+Authors@R: c(
+    person(
+        given = "Pratik",
+        family = "Gupte",
+        role = c("aut", "cre", "cph"),
+        email = "pratik.gupte@lshtm.ac.uk",
+        comment = c(ORCID = "0000-0001-5294-7819")
+        ),
+    person(
+        given = "Adam",
+        family = "Kucharski",
+        email = "adam.kucharski@lshtm.ac.uk", 
+        role = c("aut", "cph"),
+        comment = c(ORCID = "0000-0001-8814-9421")
+        ),
+    person(
+        given = "Tim",
+        family = "Russell",
+        email = "timothy.russell@lshtm.ac.uk", 
+        role = c("aut", "cph"),
+        comment = c(ORCID = "0000-0001-5610-6080")
+        ),
+    person(
+        given = "Joshua W.",
+        family = "Lambert",
+        role = c("rev"),
+        email = "joshua.lambert@lshtm.ac.uk",
+        comment = c(ORCID = "https://orcid.org/0000-0001-5218-3046")
+        ),
+    person(
+        given = "Hugo",
+        family = "Gruson",
+        role = c("rev"),
+        email = "hugo.gruson@data.org",
+        comment = c(ORCID = "https://orcid.org/0000-0002-4094-1476")
+        ),
+    person(
+        given = "Tim",
+        family = "Taylor",
+        role = c("rev"),
+        email = "tim.taylor@hiddenelephants.co.uk",
+        comment = c(ORCID = "https://orcid.org/0000-0002-8587-7113")
+        )
+    )
+Description: Estimate the severity of a disease and under-reporting of cases, as discussed in Nishiura et al. (2009) <doi:10.1371/journal.pone.0006852>.
+License: MIT + file LICENSE
+URL: https://github.com/epiverse-trace/cfr, https://epiverse-trace.github.io/cfr/
+BugReports: https://github.com/epiverse-trace/cfr/issues
+Encoding: UTF-8
+Roxygen: list(markdown = TRUE)
+RoxygenNote: 7.2.3
+Imports:
+    checkmate,
+    stats
+Suggests: 
+    dplyr,
+    tidyr,
+    purrr,
+    scales,
+    forcats,
+    ggplot2,
+    pak,
+    testthat (>= 3.0.0),
+    usethis,
+    knitr,
+    bookdown,
+    rmarkdown,
+    spelling,
+    epiparameter,
+    incidence2,
+    outbreaks
```

Comment:
From a quick glance, it doesn't look like all those packages need to be listed here. In particular, pak and usethis probably shouldn't be here. Could you review this?

Reply:
Removed pak, usethis, and rmarkdown, which I think are the only extra packages here. @jamesmbaazam has suggested replacing Tidyverse packages with {tidyverse}, but I would prefer to have them listed separately.